spec/rails_helper.rb

```diff
@@ -0,0 +1,69 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+# Add additional requires below this line. Rails is not loaded until this point!
+require 'devise'
+require 'capybara/rails'
+require 'capybara/poltergeist'
+Capybara.javascript_driver = :poltergeist
+require 'paperclip/matchers'
+require 'support/database_cleaner'
+require 'support/factory_girl'
+require 'support/devise'
+require 'support/controller_macros'
+require 'support/warden'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.